apps/meteor/client/views/account/tokens/AccountTokensTable/AccountTokensTable.tsx

```diff
@@ -123,7 +123,7 @@ const AccountTokensTable = (): ReactElement => {
 			<Box display='flex' justifyContent='center' alignItems='center' height='100%'>
 				<States>
 					<StatesIcon name='warning' variation='danger' />
-					<StatesTitle>{t('Something_Went_Wrong')}</StatesTitle>
```

Comment:
I can still find `Something_Went_Wrong` key in some translation files. They should be renamed or deleted.

Reply:
Actually, there were 2 keys having the same translation, so I removed one of the key from all translation files and used the other key at every place, removed `Something_Went_Wrong` key.